cassandra3/src/main/java/zipkin2/dependencies/cassandra3/CassandraDependenciesJob.java

```diff
@@ -125,8 +125,7 @@ public Builder localDc(@Nullable String localDc) {
       return this;
     }
 
-    /** @see {@code StorageComponent.Builder#strictTraceId(boolean)} */
-    public Builder strictTraceId(boolean strictTraceId) {
+    Builder strictTraceId(boolean strictTraceId) {
```

Comment:
only used in tests.. the code ref (and even the see ref) were causing javadoc problems.